executor/runtime/types/container.go

```diff
@@ -810,6 +826,14 @@ func (c *TitusInfoContainer) MetatronCreds() *titus.ContainerInfo_MetatronCreds
 	return c.titusInfo.GetMetatronCreds()
 }
 
+func (c *TitusInfoContainer) EffectiveNetworkMode() string {
```

Comment:
I'd just add a comment, even though there is a more detailed comment on `computeEffectiveNetworkMode` of what this means.